### PR TITLE
Fix Storage docs

### PR DESCRIPTION
**Pull request by taljaards**
## Summary
Updates some broken examples, and other misc formatting.

I just did these quickly and didn't immediately see why the example [here](https://prefecthq.github.io/prefect-gcp/cloud_storage/#prefect_gcp.cloud_storage.cloud_storage_upload_blob_from_file) isn't color-formatted.

## Relevant Issue(s)
n/a

## Checklist
**I think not really needed ->**
- [ ] Summarized PR's changes in [CHANGELOG.md](https://github.com/PrefectHQ/prefect-gcp/blob/main/CHANGELOG.md)

### CODE REVIEW

File: prefect_gcp/cloud_storage.py [hunk 1]
```diff
@@ -436,7 +436,7 @@ def example_cloud_storage_copy_blob_flow():
     logger = get_run_logger()
     logger.info(
         "Copying blob named %s from the %s bucket to the %s bucket",
-        source_blob,
+        dest_blob or source_blob,
```
Comment (desertaxle) [inline prefect_gcp/cloud_storage.py:439]: I think this should still be just `source_blob`. If you want to expand the log message to include what the name of the destination blob will be, I think that makes sense.
```suggestion
        source_blob,
```

**Review by desertaxle — CHANGES_REQUESTED** [review]
Thanks! One small comment from me. 

I think the example that doesn't have highlighting is because `python` is missing after the three backticks (\`\`\`). Would you mind adding it?

File: prefect_gcp/cloud_storage.py (hunk 1 above)
Comment (taljaards) [reply prefect_gcp/cloud_storage.py:439]: You're 100% correct! I misread the log.

**Review by desertaxle — APPROVED** [review]
Thanks!

**Pull request merged**